Telerik.Examples.Mvc/Telerik.Examples.Mvc/Views/Grid/Grid_SignalR.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿<script src="https://unpkg.com/@@aspnet/signalr@1.1.0/dist/browser/signalr.js"></script>
```

Comment:
There is an already included static resource for SignalR. Thus, I recommend updating the injected script as follows:
```
<script src="~/js/signalr/dist/browser/signalr.min.js"></script>
```